feat(components/layout): remove deprecated fluid grid gutter size options

Blackbaud-ErikaMcVey opened the pull request:
BREAKING CHANGE: This change removes the `SkyFluidGridGutterSize` enum and the numerical options (0,
1, 2) from `SkyFluidGridGutterSizeType`. To address this, only use the strings 'small', 'medium',
and 'large' for  the fluid grid component's `gutterSize` property, and use
`SkyFluidGridGutterSizeType` for Typescript typing.

blackbaud-sky-build-user commented:
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/585/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/585/storybooks/ag-grid-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/585/storybooks/layout-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/585/storybooks/pages-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/585/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/585/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/585/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/585/playground/)

Blackbaud-ErikaMcVey committed:
feat(components/layout): remove deprecated fluid grid gutter size options

BREAKING CHANGE: This change removes the `SkyFluidGridGutterSize` enum and the numerical options (0,
1, 2) from `SkyFluidGridGutterSizeType`. To address this, only use the strings 'small', 'medium',
and 'large' for  the fluid grid component's `gutterSize` property, and use
`SkyFluidGridGutterSizeType` for Typescript typing.
